### DISCUSSION TITLE
File open picker does not work on all platforms

### DISCUSSION THREAD

**Discussion by garawaa** [unoplatform/uno#10923] [Q&A]
File open picker does not work on all platforms

**Comment by MartinZikmund** [comment]
@garawaa Can you give us more details please? Provide some code if possible

**Reply by garawaa** [reply]
```
        public async void OnBtnClick(object sender, RoutedEventArgs e)
        {
            var fileOpenPicker = new FileOpenPicker();
            fileOpenPicker.SuggestedStartLocation = PickerLocationId.ComputerFolder;
            fileOpenPicker.FileTypeFilter.Add(".txt");
            fileOpenPicker.FileTypeFilter.Add(".csv");
            StorageFile pickedFile = await fileOpenPicker.PickSingleFileAsync();
        }
```
I just copied it from your doc.
This code does not work. Working on android and wasm first time but after rebuild project it also crashes.  Compiled without an error. But when click on button to open picker the app just crashes. My OS windows 11 home edition with latest update. Tool: visual studio 2022 CE prev. all uno tools updated
